packages/__globalPreview/frontend.html

```diff
@@ -39,10 +39,17 @@
           ></div>
 
+          <!-- <div
+            data-hydration-component="metadata-fields"
+            data-component="metadata-fields"
+            data-hydration-props='{"type":"Featured","authors":[{"asset_name":"JB Christy","asset_assetid":"128503"}],"producers":[{"asset_name":"JB Christy","asset_assetid":"128503"}],"writers":[{"asset_name":"JB Christy","asset_assetid":"128503"}],"editors":[{"asset_name":"JB Christy","asset_assetid":"128503"}],"videographers":[],"photography":[],"media":[{"asset_name":"Arnie Schwartz","asset_assetid":"128188","asset_metadata_personEmail":"aponomarenko@squiz.net"},{"asset_name":"Jane Doe","asset_assetid":"127752","asset_metadata_personEmail":"jane.doe@stanford.edu.au"}],"campus":{"asset_name":"Stanford BeWell","asset_assetid":"28337","asset_url":""},"related":[{"asset_assetid":"128103","asset_name":"Cardinal at Work","asset_url":"","target_id":""},{"asset_assetid":"28433","asset_name":"Campus beauty","asset_url":"https://sug-web.matrix.squiz.cloud/report/university-news/campus-and-facilities/campus-beauty","target_id":"125110"}]}'
+          ></div> -->
+
+          <!-- No Media Contacts -->
           <div
             data-hydration-component="metadata-fields"
             data-component="metadata-fields"
-            data-hydration-props='{"type":"Basic","authors":[{"asset_name":"JB Christy","asset_assetid":"128503"}],"producers":[{"asset_name":"JB Christy","asset_assetid":"128503"}],"writers":[{"asset_name":"JB Christy","asset_assetid":"128503"}],"editors":[{"asset_name":"JB Christy","asset_assetid":"128503"}],"videographers":[],"photography":[],"media":[{"asset_name":"Arnie Schwartz","asset_assetid":"128188","asset_metadata_personEmail":"aponomarenko@squiz.net"},{"asset_name":"Jane Doe","asset_assetid":"127752","asset_metadata_personEmail":"jane.doe@stanford.edu.au"}],"campus":{"asset_name":"Stanford BeWell","asset_assetid":"28337","asset_url":""},"related":[{"asset_assetid":"128103","asset_name":"Cardinal at Work","asset_url":"","target_id":""},{"asset_assetid":"28433","asset_name":"Campus beauty","asset_url":"https://sug-web.matrix.squiz.cloud/report/university-news/campus-and-facilities/campus-beauty","target_id":"125110"}]}'
+            data-hydration-props='{"type":"Featured","authors":[{"asset_name":"JB Christy","asset_assetid":"128503"}],"producers":[{"asset_name":"JB Christy","asset_assetid":"128503"}],"writers":[{"asset_name":"JB Christy","asset_assetid":"128503"}],"editors":[{"asset_name":"JB Christy","asset_assetid":"128503"}],"videographers":[],"photography":[],"media":[],"campus":{"asset_name":"Stanford BeWell","asset_assetid":"28337","asset_url":""},"related":[{"asset_assetid":"128103","asset_name":"Cardinal at Work","asset_url":"","target_id":""},{"asset_assetid":"28433","asset_name":"Campus beauty","asset_url":"https://sug-web.matrix.squiz.cloud/report/university-news/campus-and-facilities/campus-beauty","target_id":"125110"}]}'
```

Comment:
Not completely sure if we want to keep all this before merging - please let me know @broleomo 😄